test/framework/log/log.go

```diff
@@ -0,0 +1,168 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// log package implements an interface similar to klogr, and reuses much
+// of the same code, but makes the output appropriate for Ginkgo
+package log
```

Comment:
In the test framework there is already a custom logger under `test/framework/clusterctl/logger`. is it possible to make these two converge and get only one?

Reply:
Taking a look now.

Reply:
I've removed it from this PR, will unify them in a separate PR